_docs/tasks/consul.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Running Istio in Docker with Consul
+overview: Make Istio run in docker environment by integrating Consul as a service registry.
+
+order: 42
+
+layout: docs
+type: markdown
+---
+
+Configure Istio to run in docker environment by integrating Consul as a service registry.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the [Installation guide](./installing-istio.html).  You will
+  need `kubectl` and Steps 1-4 from `Installation step` only for this Task.
+
```

Comment:
should we say `kubectl` 1.7.3 or later?

Reply:
If there is a version dependency, I would hope it would be in the installing-istio.html page - right?

Reply:
ok that sounds good.